core/src/main/java/org/elasticsearch/index/get/GetField.java

```diff
@@ -90,4 +95,58 @@ public void writeTo(StreamOutput out) throws IOException {
             out.writeGenericValue(obj);
         }
     }
+
+    @Override
+    public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
+        builder.startArray(name);
+        for (Object value : values) {
+            builder.value(value);
+        }
+        builder.endArray();
+        return builder;
+    }
+
+    public static GetField fromXContent(XContentParser parser) throws IOException {
+        if (parser.currentToken() != XContentParser.Token.FIELD_NAME) {
+            throw new ParsingException(parser.getTokenLocation(),
```

Comment:
will do once you pushed your PR, will reuse those methods that you are introducing